pandas/tests/series/test_arithmetic.py

```diff
@@ -832,6 +832,40 @@ def test_scalarop_preserve_name(self, datetime_series):
         assert result.name == datetime_series.name
 
 
+class TestInplaceOperations:
+    @pytest.mark.parametrize(
+        "dtype1, dtype2, dtype_expected, dtype_mul",
```

Comment:
can you re-write this to be a list-of-tuples, it is non-obvious what you are testing here because of the translation.
IOW each case is easily read on a single tuple

Reply:
Right, done